cli/azd/test/functional/testdata/samples/storage-bicepparam/infra/main.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This will be unused because there is a `main.bicepparm` file here, right?  Should we delete?